### PR TITLE
Nobody cared who I was until I put on the mask

### PR DESCRIPTION
**Pull request by LOLSKELETONS**
Adds bane gear to the code, currently only obtainable via adminbus

I've already tested it a few times and it seems to work.

### CODE REVIEW

**Comment by Kokojos** [comment]
Why not.

**Comment by Zadeon** [comment]
Good stuff here. Gives lolskeletons the basics of how all this stuff works too.

**Comment by carlarc** [comment]
yes

**Comment by carlarc** [comment]
make it a syndie kit too

File: code/modules/clothing/under/miscellaneous.dm [hunk 1]
```diff
@@ -1,3 +1,13 @@
+
+/obj/item/clothing/under/banearmour
+    name = "banearmour"
+    desc = "You merely adopted the darknes,I was born in it.."
```
Comment (JohnGinnane) [inline code/modules/clothing/under/miscellaneous.dm]: "darknes" should be "darkness"

File: code/modules/clothing/suits/armor.dm [hunk 2]
```diff
@@ -1,3 +1,16 @@
+
+/obj/item/clothing/suit/armor/banecoat
+    name = "banecoat"
+    desc = "When Gotham is ashes, you have my permission to die."
```
Comment (ghost) [inline code/modules/clothing/suits/armor.dm]: When Gotham is ashes, then you have my permission to die.

**Comment by ghost** [comment]
If this doesn't let you do the backbreaker from wrestling, I'm fucking out.

File: code/modules/clothing/under/miscellaneous.dm (hunk 1 above)
Comment (ghost) [inline code/modules/clothing/under/miscellaneous.dm]: Also, an ellipsis should have three dots and another one if they're the end of a sentence.

File: code/modules/clothing/suits/armor.dm [hunk 3]
```diff
@@ -1,7 +1,7 @@
 
 /obj/item/clothing/suit/armor/banecoat
     name = "banecoat"
-    desc = "When Gotham is ashes, you have my permission to die."
+    desc = "When Gotham is in ashes,then you have my permission to die."
```
Comment (ghost) [inline code/modules/clothing/suits/armor.dm]: > Removed the space
> not good shit no siree

File: code/modules/clothing/under/miscellaneous.dm [hunk 4]
```diff
@@ -1,7 +1,7 @@
 
 /obj/item/clothing/under/banearmour
     name = "banearmour"
-    desc = "You merely adopted the darknes,I was born in it.."
+    desc = "You merely adopted the darkness,I was born in it"
```
Comment (ghost) [inline code/modules/clothing/under/miscellaneous.dm]: Needs a space after the comma and a period at the end now

**Comment by ghost** [comment]
Good shittu. +1

**Comment by Nebyab** [comment]
ok

**Pull request merged**